image.go

```diff
@@ -20,6 +20,8 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// Image provides access to internal image info
```

Comment:
FWIW that's what I was thinking too.

I'm also curious about the need to know such things. @abhinandanpb suggested in https://github.com/containerd/containerd/pull/1311#issuecomment-321328126 that it was so cri-containerd could keep it in an image store (and pointed me to [the code](https://github.com/abhinandanpb/cri-containerd/blob/0b3953c3bf8c3cceed594f008b793c4033dea657/pkg/server/helpers.go)) but it isn't clear to me why cri-containerd needs to store a duplicate of this information itself at all rather than relying on containerd's store as a source of truth. Maybe there is a facet of the CRI interfaces I don't understand which requires it?